src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3356,3 +3356,231 @@ public abstract partial class WaitHandle : System.IDisposable
         public virtual bool WaitOne(System.TimeSpan timeout) { return default(bool); }
     }
 }
+
+namespace System.Globalization
```

Comment:
There already is a `namespace System.Globalization` declaration in this file, you should add the missing types in that one.